packages/app-builder-lib/src/util/packageDependencies.ts

```diff
@@ -3,7 +3,7 @@ import { executeAppBuilderAsJson } from "./appBuilder"
 
 export function createLazyProductionDeps(projectDir: string, excludedDependencies: Array<string> | null) {
   return new Lazy(async () => {
-    const args = ["node-dep-tree", "--dir", projectDir]
+    const args = ["node-dep-tree", "--flatten", "--dir", projectDir]
```

Comment:
Is this a new arg `--flatten` you've implemented in app-builder or something that already exists? Curious as to what changes you made to app-builder for this to work? 🙂

Reply:
Great idea! Love it